docs/repo/crates/discv4.md

```diff
@@ -192,9 +192,8 @@ Once the `Discv4Service::new()` function completes, allowing the `Discv4::bind()
 ## Polling the Discv4Service and Discovery Events
 In Rust, the owner of a [`Future`](https://doc.rust-lang.org/std/future/trait.Future.html#) is responsible for advancing the computation by polling the future. This is done by calling `Future::poll`.
 
-Lets take a detailed look at how `Discv4Service::poll` works under the hood. This function has many moving parts, so we will break it up into smaller sections. If you would like to check out the function in it's entirety, you can [click here]().
+Lets take a detailed look at how `Discv4Service::poll` works under the hood. This function has many moving parts, so we will break it up into smaller sections. If you would like to check out the function in it's entirety, you can [click here](https://github.com/paradigmxyz/reth/blob/main/crates/net/discv4/src/lib.rs#L1302).
```

Comment:
```suggestion
Lets take a detailed look at how `Discv4Service::poll` works under the hood. This function has many moving parts, so we will break it up into smaller sections.
```
I'd just remove the link to the function itself since code structure may change, I think readers will be able to find the function if they want to